packages/bitcoin/src/bip322/sign-message-bip322.ts

```diff
@@ -0,0 +1,54 @@
+import * as btc from '@scure/btc-signer';
+import { hexToBytes } from '@stacks/common';
+
+import { hashBip322Message } from './bip322-utils';
+
+// TODO: Complete this fn
```

Comment:
Lets add some context when moving stuff like this.

In the `extension` repo, in the history we can see the PR this was added and the ticket number:
<img width="756" alt="Screenshot 2023-11-20 at 09 22 11" src="https://github.com/leather-wallet/mono/assets/2938440/43d57b79-4e90-41da-9056-f10e2ff79e35">

When migrating stuff like this I think it's a good idea to update the comments with some of this information so we can keep the context in the new repo

Reply:
I tried reading through the history but still don't understand much context on that part. @kyranjamie can you, please, help us our on this one? (tagging you because of git blame :D )

Reply:
This comment relates to issues ran into between the two different btc libraries. I understand Pete is just asking you to reference this in the commit msg?

Reply:
Yeah I just think we should add the ticket number in the comment so we can trace in case we ever need to